### ISSUE TITLE
building problems from subfiles while using \import in root

### ISSUE THREAD

**Issue by dslemusp** [James-Yu/LaTeX-Workshop#723]
### Description

My project file structure is as follows: 

- root.tex
- Chapters/Chapter-1/
  - chapter-1.tex
  - Figs
  - Tex
    - 1-Introduction.tex

As I have several subfolders i wanted to use `\import` or `\subimport` to call the chapter files from `root.tex`, such that I can for instance use `\include{Tex/1-Introduction}` in `chapter-1.tex` without having to use the whole relative path from the `root.tex` file.

Doing this works fine as soon as build it from `root.tex`. However if I try to build it from any of the included files (i.e. `chapter-1.tex` or `1-Introduction.tex`) it doesn't do anything, not even an error or warning. I suspect there might be an issue on how the root is found from the subfiles when the `\subimport` or `\import` command is used in the root. Is important to note that if I use `\include` in the root everything works fine, but It requires to have the whole long relative path hardcoded in the subfiles `\include` calls.

### Log messages

None generated

### How to reproduce

[mwe.zip](https://github.com/James-Yu/LaTeX-Workshop/files/2235182/mwe.zip)

1. Decompress the attached mwe.zip file
2. Open folder with vscode. VSCode->File->Open Folder -> mwe
3. Try to build the project from chapter-1.tex or 1-Introduction.tex (Doesn't do anything)
4. Comment line 7 and uncomment line 10 in root.tex. Comment line 6 and uncomment line 9.
5. Try to build again from any of the subfiles (Now it works)

### Expected behavior

Step 3 should be able to build the project from any subfile as it does in Step 5

### Actual behavior

Step 3. should not work, no output log, nothing happens.
Step 5. should build.

### Additional Information

User settings:
```
"latex-workshop.latex.tools": [

        {
            "name": "latexmk",
            "command": "latexmk",
            "args": [
                "-interaction=nonstopmode",
                "-file-line-error",
                "-pdfps",
                "%DOC%"
            ]
        }
    ]
```

**Comment by jlelong** [comment]
Thanks for the detailed minimal example. On it!

**Comment by dslemusp** [comment]
@jlelong Thank you very much for addressing this so quickly. I was wondering whether the feature is already implemented in the current version (as today 5.7.0), as although you clearly solved the issue I still have the same problem.

Thank you again

**Comment by James-Yu** [comment]
@dslemusp No, but I will try to make a release in the following day with all fixes in the interim.
